packages/fast-components-class-name-contracts-msft/src/select/index.ts

```diff
@@ -8,4 +8,19 @@ export interface SelectClassNameContract extends BaseSelectClassNameContract {
      * The glyph shown in the select toggle button
      */
     select_toggleGlyph?: string;
+
+    /**
+     * The select toggle button
+     */
+    select_button?: string;
+
+    /**
+     * The select toggle button content region
+     */
+    select_button_contentRegion?: string;
```

Comment:
I think this would be select_buttonContentRegion

Reply:
yep